docs/setting-your-IDE.md

```diff
@@ -0,0 +1,92 @@
+## The basics
+
+If you are interested in being able to quickly debug and develop Mailchimp Open Commerce, consider installing these useful tools to help you.
+
+- You should use [nvm](https://github.com/nvm-sh/nvm) to install and manage NodeJS.
+- Visual Studio Code is a preferred text editor but you can use Sublime Text, Emacs, Vim or any other editor of your choice.
+- [Kadira](https://github.com/kadira-open/kadira-server) can help you optimize performance of a Meteor app.
```

Comment:
Let's remove Kadira. Also it's only needed for meteor app, which at this point is only Admin.